### DISCUSSION TITLE
Report allows alert a error message:Report Schedule is still working, refusing to re-compute.

### DISCUSSION THREAD

**Discussion by DEVLHK** [apache/superset#18331] [Q&A / Help] [answered]
## Screenshot
![image](https://user-images.githubusercontent.com/42260778/137462746-f59b43d1-3d24-4780-9350-ca91a3254fe4.png)

[drag & drop image(s) here!]

## Description
Report always alert a error message:Report Schedule is still working, refusing to re-compute.

## Design input
[describe any input/collaboration you'd like from designers, and
tag accordingly. For design review, add the
label `design:review`. If this includes a design proposal,
include the label `design:suggest`]

**Comment by mayurnewase** [comment]
Will need more info.
What are reports for?
Chart/ Dashboard -> how big they are and how long does it take to query it?
And all your config from config.py related to reports and alerts.
Any info on terminal?
Way to reproduce on default chart/dashboard.

**Comment by mayurnewase** [comment]
Also you can kickstart it again from initial state by updating report_schedule table -> your schedule entry -> last_state column to `Not triggered` in your metadata database.

**Comment by KarthikGM05** [comment]
I have the same problem. And after a while I get `[Errno 110] Connection timed out`
![image](https://user-images.githubusercontent.com/46784229/140871667-c0979db7-d178-4af2-b687-6b38afc2f78b.png)
Any ideas?

**Comment by Torry29** [comment]
I have same issue and how did u solve it?

**Comment by PriyaGanji** [comment, marked as answer]
if the status of Alert/Report is in working  then this error is coming. So to solve this disable the report/alert and again activate it.
it will work. 
so to test this first change the status "working" to "error" in db  then run, it is working in my case.

Hopefully it will work for you also.

**Reply by iltafah** [reply]
This solved the problem, disabled the report waited for about 2 minutes just in case, and then reenabled it, it works now

**Comment by ghost** [comment]
Hi @iltafah ,
How did it resolved, can you brief it?
